bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -408,7 +408,7 @@
 		<item-type>Number:Time</item-type>
 		<label>Momentary Duration</label>
 		<description>Set the output relay closure duration for momentary relay modes (A-C).</description>
-		<state min="0.2" max="25" step="0.1" pattern="%.1f %unit%"/>
+		<state min="0.1" max="6300" step="0.1" pattern="%.1f %unit%"/>
```

Comment:
Not 100% sure, but this might require update instructions. Could you verify?

Reply:
I'm pretty sure an `update-channel` would do. But you can inspect the json db or just observe the channel definition in the UI. Ultimately it is also possible to remove + add the channel. but don;t think that is needed.

Reply:
> I'm pretty sure an `update-channel` would do.

I was able to confirm in the [`openhab-core`](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/update/UpdateChannelInstructionImpl.java#L60).

Since the binding is basically creating the device channels for a given thing when they weren't available previously, I could update the implementation to update existing channels each time the thing handler is initialized.

However, channel labels and descriptions can be customized by the users which is more predominant when using textual config files. I actually added a fix for the legacy implementation for that specific case last month. I am just wondering if these specific attributes should be updated in the end.

Reply:
Change included in #18698